### PR TITLE
KMS Scanner Documentation

### PR DESCRIPTION
**Pull request by red2k18**

### CODE REVIEW

File: _docs/_latest/configure/scanner/rules.md [hunk 1]
```diff
@@ -943,3 +943,43 @@ rules:
...
+  
+* `mode`
+  * **Description**: Choose whether or not the list of values will be
+    interpreted as a blacklist or not.
+  * **Valid values**: String.  Only `blacklist` mode is supported.
+
+* `resource`
+  * `type`
+    * **Description**: The type of the resource.
+    * **Valid values**: String, `organization`.
```
Comment (joecheuk) [inline _docs/_latest/configure/scanner/rules.md]: Can you be more specific? e.g. Only `organization` is supported.

File: _docs/_latest/configure/scanner/descriptions.md [hunk 2]
```diff
@@ -234,3 +234,11 @@ Allow customers to ensure the retention policies on their resources are set as i
 
 For examples of how to define scanner rules for retention, see the ['retention_rules.yaml'](https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/rules/retention_rules.yaml) rule file.
 
+## KMS scanner
+
+This scanner checks if the crypto keys in the organization are rotated within 
+the time specified, and notifies if they are not. 
+
+For examples of how to define scanner rules for your crypto keys, see the
+[`kms_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/rules/kms_rules.yaml)
```
Comment (joecheuk) [inline _docs/_latest/configure/scanner/descriptions.md]: This is not the right link, can we use dev instead of stable

Comment (red2k18) [reply _docs/_latest/configure/scanner/descriptions.md]: Sure, done.

File: _docs/_latest/configure/scanner/rules.md (hunk 1 above)
Comment (red2k18) [reply _docs/_latest/configure/scanner/rules.md]: Done

**Pull request merged**